xonsh/proc.py

```diff
@@ -336,9 +342,67 @@ def wrapped_simple_command(args, stdin, stdout, stderr):
                         stderr.write(r[1])
                 elif r is not None:
                     stdout.write(str(r))
-                return True
+                return 0  # returncode for succees
```

Comment:
I was hoping to use `True` and `False` for these values for the `and`/`or` business, but we can discuss that once I've gotten back into gear and finished that off...